pan_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: pan_joint
    joint_speed: 0.25
    motor:
        id: 1
        init: 2048
        min:  0
        max:  4095

tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: tilt_joint
    joint_speed: 0.25
    motor:
        id: 2
        init: 2048
        min: 0
        max: 4095
